lib/src/main/java/com/google/android/material/composethemeadapter/MdcTheme.kt

```diff
@@ -70,6 +73,7 @@ fun MdcTheme(
     readTypography: Boolean = true,
     readShapes: Boolean = true,
     setTextColors: Boolean = false,
+    setDefaultFontFamily: Boolean = false,
```

Comment:
I'm not sure this really needs to be a parameter. Assume true always?

Reply:
That would be nice, however there's fairly complex order of precedence logic that the framework seems to use. It was getting a bit much to implement/test so a parameter seemed like a good middle ground.